hangman/lib/hangman/game.ex

```diff
@@ -0,0 +1,113 @@
+defmodule Hangman.Game do
+    
+  def new_game() do
+    %Hangman.Tally{}
+      |> Map.put(:letters, get_word())
+  end
+
+  defp get_word() do
+    Dictionary.random_word 
+      |> String.codepoints()
+      |> Enum.map(&( "_#{&1}" ))
+  end
+
+  def tally(%Hangman.Tally{} = game) do
+    %{
+      game_state: game.game_state,
+      turns_left: game.turns_left,                
+      letters:    game.letters |> format_letters(),              
+      used:       game.used,   
+      last_guess: game.last_guess           
+    }
+  end
+
+  def tally(_) do
```

Comment:
I'm not6 deducting anything, but the Elixir/Erlang style is not to add these catch-all functions. Instead, we just let the code fail for itself. That has the added benefit of showing the actual arguments passed.